### ISSUE TITLE
fix(model): restore committed compute dependencies for Qwen package build

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#12038]
Committed internal/model references compute symbols absent from committed trunk, preventing unrelated gateway regression execution.

Observed at 96e6b498f8218c16856558da052b6004039616fd in an isolated managed worker: package compilation fails for compute.TiledConvConcatForwardSlices, compute.QSADynamicGatingThreshold, and compute.QSAPerplexityDeltaTolerance in internal/model/qwen35.go. All three references were introduced by f9a8c6788d7186298e9815b499dc77e7448b2637. Their definitions currently exist only in uncommitted companion files within internal/compute; tests must not silently import those changes as committed evidence.

For: native serving developers and gateway maintainers. Problem: committed package graph is incomplete. Today: gateway HTTP regression cannot compile. Better because: atomically committing the reviewed dependency contract, or a minimal reviewed caller correction, restores reproducible validation. Centrality: Enabling; native serving priority 2. Scope is the existing three-symbol internal/model-to-internal/compute contract, without new compute features.

Done condition: from an isolated committed checkout, run go test -count=1 ./internal/model ./internal/gateway and capture the actual successful package execution; confirm no peer working-tree files are imported. Use the supported WSL test runner on Windows. Preserve existing numerical semantics and obtain independent review before landing dependencies.

This blocks the real HTTP witness for #11423 during trajectory audit #11294. Commit f9a8c6788 cites #469, but that closed issue describes grammar-constrained tool decoding and does not track this build defect. Keep reconciliation linked here instead of treating that unrelated issue as the dependency tracker.

**Comment by anthony-chaudhary** [comment]
Independently reproduced while validating #12035: `fak validate --mine internal/gpulease/lease.go --mine internal/gpulease/shared_test.go --mine internal/flock/flock_unix.go --mine internal/flock/flock_windows.go` on isolated base 96e6b498f8218c16856558da052b6004039616fd fails build/vet with exactly these three missing compute symbols. The two owned packages pass complete WSL tests and vet independently; the reviewed shared/exclusive resource patch is held without landing rather than incorporating peer WIP. This is another dependent validation blocker, not a separate defect.

**Comment by anthony-chaudhary** [comment]
The three missing symbols are now present in local committed source `81b0d855ab92140a5bceb5f34f4ef1c3aec24293`. An isolated checkout of that revision builds and vets `./internal/model ./internal/gateway` without a shim or peer working-tree files.

Actual numerical execution still fails, so this issue's successful-package condition is not yet met:

```text
go test -count=1 -run '^TestQwen35LinearAttnBatchedMatchesScalar$' ./internal/model
qwen35_chunked_test.go:44: layer 0 seq 1 tok 0[0] = 3abe8dd8, want 3abe8dd7 (0.0014538122 vs 0.0014538121)
```

The newly connected `TiledDepthwiseConv1DChannelMajor` uses `compute.Silu`, whose intermediate precision differs from canonical `model.silu`. A standalone CPU counterexample at input `2` gives canonical bits `3fe17bea` versus tiled bits `3fe17beb`. A bounded correction will preserve the existing model test and change only the tiled convolution's rounding, with a golden-value regression for both kernel-size branches. Global `compute.Silu` and GPU shaders remain outside that correction.

The independent availability-setting defect is tracked by #12049. Keep this issue open until its full stated witness succeeds; build/vet alone and the separate physical GDN mixer witness do not establish that result.

**Comment by anthony-chaudhary** [comment]
Fresh blocker evidence from #12346 guarded commit validation: the exact sampled-endpoint patch is green in its focused suite and vet, but the full internal/model gate remains red on TestQwen35PrefillAndStepCPUContinuation (Prefill logits[0] NaN), TestQwen35LinearAttnBatchedResumesState (1-bit mismatch), TestQwen35LinearAttnBatchedMatchesScalar, and TestQwen35ChunkedBatchedMatchesScalar (batched/scalar bit mismatches). This extends the already-recorded numerical execution failure here and currently blocks #12346 from a guarded commit. Source receipt: /Users/anthony/Documents/GitHub/fak/_scratch/issue-12346/commit.json; #12346 receipt: https://github.com/anthony-chaudhary/fak/issues/12346#issuecomment-5592279578. Keep diagnosis/fix isolated from the preserved #12346 patch.

**Comment by anthony-chaudhary** [comment]
Bounded numerical investigation and fix receipt (not yet landed).

Red-before on clean committed ae2151e06: the three parity witnesses failed (`3a046741` vs `3a046740`; `b84597c0` vs `b84597b0`). Root cause is local to `TiledDepthwiseConv1DChannelMajor`: both branches used float64-ordered `compute.Silu` rather than the scalar model rounding order. The candidate changes only `internal/compute/wave32_kda.go` and its test, leaves global `compute.Silu` and shaders unchanged, and pins both K=4/general branches at `SiLU(2)=0x3fe17bea`.

The continuation NaN is independent: that fixture has no linear-attention layers and omitted `RopeTheta`, causing `0*Inf` in RoPE. It is separately tracked/fixed locally under #12433. The unrelated intermittent Darwin timing witness is tracked as #12437.

Green candidate receipts with task-owned caches:
- focused tiled golden and all four blocker tests: PASS
- full `internal/compute`: PASS
- build and vet for compute/model/gateway: PASS
- literal issue witness `go test -count=1 ./internal/model ./internal/gateway`: PASS (`model 403.667s`, `gateway 88.551s`)
- independent adversarial review: `CONFIRMED_VALID`; independent tester: `CLAIM_TEST_GREEN`

Candidate base modules are `internal/compute@r406+g01fa5c4bc` and `internal/model@r711+gae2151e06`; there is deliberately no changed module@rev yet. Guarded explicit-path commit on current trunk base d19e77b43 refused before staging with `COMMITTED_RED` solely because automatic Strix discovery failed with SSH exit 255 (#12382). No bypass flag was used; nothing was committed or pushed. Main is also temporarily behind origin with unrelated dirty macbench collisions, so unsafe reconciliation was not attempted.

Keep #12038 open. Exact retry: restore/reuse this three-file candidate on a synchronized trunk, make the required Strix validation reachable (or land a sanctioned selector fix that still preserves applicable device evidence), rerun the literal model+gateway witness, then guarded explicit-path commit/push. Only the resulting pushed compute module@rev may close this issue.
